etcd/getting-started-with-etcd.md

```diff
@@ -6,6 +6,36 @@ Application containers running on your cluster can read and write data into etcd
 
 <a class="btn btn-default" href="../latest/api.html">Complete etcd API Docs</a>
 
+## Setting up etcd
+
+Container Linux comes pre-installed with an `etcd-wrapper` script which can be configured to **run etcd from inside of a container**. Container Linux also comes pre-installed with an `etcd-member.service` systemd unit which by default uses the etcd3 image. This means you to use etcd3 without needing to install the etcd3 binary.
+
+To enable the `etcd-member.service` unit you need to manually disable the `etcd2.service` unit and enable the `etcd-member.service`.
```

Comment:
(avoid 2nd person) "To enable the etcd v3 service, first ensure that `etcd2.service` is disabled. Then, enable and start the `etcd-member.service`:"

(& fact check etcd2.service default status, potentially drop step 1)